http-server/apache2-web-server/README.md

```diff
@@ -137,43 +160,43 @@ $ singularity instance stop httpd
 <br>
 
 
-### Building the contianer from a definition file:
+## Building the container from a definition file:
 
 To build the container from a recipe, you will need root access, and the definition file.
 
-First, make the definition file:
+First, download the [definition file](https://raw.githubusercontent.com/WestleyK/examples/master/http-server/apache2-web-server/httpd.def):
 
 ```
-$ nano httpd.def
-```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/httpd.def
 ```
-Bootstrap: docker
-From: httpd:latest
-
-%post
-    # Change the port we are listening on to 8080 instead of 80
-    sed -ie "s/^\(Listen\).*/\1 8080/" /usr/local/apache2/conf/httpd.conf
 
-%startscript
-    httpd
-```
+<br>
 
-Or you can download it:
+Then to build the container:
 
 ```
-$ wget https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/httpd.def
+$ sudo singularity build httpd.sif httpd.def
 ```
 
 <br>
 
-Then to build the container:
+Or use remote builder:
+
+You do not need root access.
```

Comment:
Maybe should be a better idea to paraphrase this like: "If you do not own root access, you can also use our remote builder"